### PR TITLE
fix(clikey): changed default`terrad` home path

### PR DESCRIPTION
**Pull request by rtviii**
# Usage example

```typescript
import {CLIKey} from '@terra-money/terra.js'

const clikey = new CLIKey({
    'keyName':'terradev',
})
console.log(clikey.accAddress);
console.log(clikey.valAddress);
```

Current implementation panics with (truncated, full error at the bottom if interested):
```
Error: Command failed: terrad keys show terradev --output json --home /home/rxz/go/bin/terrad
Error: couldn't get client config: Config File "client" Not Found in "[/home/rxz/go/bin/terrad/config]"
```
With changed path: 
```typescript
terra1hw4twm4kajhjpl6qapg0pn2cdtpnagp65planh
terravaloper1hw4twm4kajhjpl6qapg0pn2cdtpnagp65wnqry
```
...having changed default home to `$HOME/.terrad/config`, which seems to be the default for the last few versions of core did the trick.  $HOME is per node's`os`. 

```bash
Error: couldn't get client config: Config File "client" Not Found in "[/home/rxz/go/bin/terrad/config]"
Usage:
  terrad keys show [name_or_address [name_or_address...]] [flags]


Error: Command failed: terrad keys show terradev --output json --home /home/rxz/go/bin/terrad
Error: couldn't get client config: Config File "client" Not Found in "[/home/rxz/go/bin/terrad/config]"
Usage:
  terrad keys show [name_or_address [name_or_address...]] [flags]

Flags:
  -a, --address                  Output the address only (overrides --output)
      --bech string              The Bech32 prefix encoding for a key (acc|val|cons) (default "acc")
  -d, --device                   Output the address in a ledger device
  -h, --help                     help for show
      --multisig-threshold int   K out of N required signatures (default 1)
  -p, --pubkey                   Output the public key only (overrides --output)

Global Flags:
      --home string              The application home directory (default "/home/rxz/.terra")
      --keyring-backend string   Select keyring's backend (os|file|test) (default "os")
      --keyring-dir string       The client Keyring directory; if omitted, the default 'home' directory will be used
      --log_format string        The logging format (json|plain) (default "plain")
      --log_level string         The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
      --output string            Output format (text|json) (default "text")
      --trace                    print out full stack trace on errors


    at checkExecSyncError (child_process.js:760:11)
    at execSync (child_process.js:833:15)
    at CLIKey.loadAccountDetails (/home/rxz/dev/TFL/create-send-tx/src/key/CLIKey.ts:47:15)
    at CLIKey.get (/home/rxz/dev/TFL/create-send-tx/src/key/CLIKey.ts:61:12)
    at Object.<anonymous> (/home/rxz/dev/TFL/create-send-tx/tx.ts:28:20)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Module.m._compile (/home/rxz/.nvm/versions/node/v14.17.6/lib/node_modules/ts-node/src/index.ts:1371:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Object.require.extensions.<computed> [as .ts] (/home/rxz/.nvm/versions/node/v14.17.6/lib/node_modules/ts-node/src/index.ts:1374:12)
    at Module.load (internal/modules/cjs/loader.js:937:32) {
  status: 1,
  signal: null,
  output: [
    null,
    Buffer(0) [Uint8Array] [],
    Buffer(1292) [Uint8Array] [
       69, 114, 114, 111, 114,  58,  32,  99, 111, 117, 108, 100,
      110,  39, 116,  32, 103, 101, 116,  32,  99, 108, 105, 101,
      110, 116,  32,  99, 111, 110, 102, 105, 103,  58,  32,  67,
      111, 110, 102, 105, 103,  32,  70, 105, 108, 101,  32,  34,
       99, 108, 105, 101, 110, 116,  34,  32,  78, 111, 116,  32,
       70, 111, 117, 110, 100,  32, 105, 110,  32,  34,  91,  47,
      104, 111, 109, 101,  47, 114, 120, 122,  47, 103, 111,  47,
       98, 105, 110,  47, 116, 101, 114, 114,  97, 100,  47,  99,
      111, 110, 102, 105,
      ... 1192 more items
    ]
  ],
  pid: 241340,
  stdout: Buffer(0) [Uint8Array] [],
  stderr: Buffer(1292) [Uint8Array] [
     69, 114, 114, 111, 114,  58,  32,  99, 111, 117, 108, 100,
    110,  39, 116,  32, 103, 101, 116,  32,  99, 108, 105, 101,
    110, 116,  32,  99, 111, 110, 102, 105, 103,  58,  32,  67,
    111, 110, 102, 105, 103,  32,  70, 105, 108, 101,  32,  34,
     99, 108, 105, 101, 110, 116,  34,  32,  78, 111, 116,  32,
     70, 111, 117, 110, 100,  32, 105, 110,  32,  34,  91,  47,
    104, 111, 109, 101,  47, 114, 120, 122,  47, 103, 111,  47,
     98, 105, 110,  47, 116, 101, 114, 114,  97, 100,  47,  99,
    111, 110, 102, 105,
    ... 1192 more items
  ]
}
```

### CODE REVIEW

**Comment by Vritra4** [comment]
cliPath? not home? :)

**Comment by rtviii** [comment]
@Vritra4 yeah exactly haha.. not sure if i'm confused or the defautls outdated, but this works.

File: src/key/CLIKey.ts
Comment (Vritra4) [inline src/key/CLIKey.ts:39]: @rtviii i mean this :) i think params.home is correct.

Comment (rtviii) [reply src/key/CLIKey.ts:39]: @Vritra4 yeah, just noticed. late night, sorry. should be `.home` not `.cliPath` on the second line. At least that's what worked.

Comment (rtviii) [reply src/key/CLIKey.ts]: See edits below :woozy_face:

**Review by Vritra4 — APPROVED** [review]

**Comment by rtviii** [comment]
Geoff i see you running tests so unless you merge after you are done, i'll merge in the morning if all's well.

**Pull request merged**
